package.json

```diff
@@ -20,7 +20,7 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "fmt": "prettier --write src/*.ts demo/*.ts *.md",
     "build": "tsc && webpack -p",
-    "demo": "du -ab node_modules/ | node build/src/cli.js --title 'node_modules for webtreemap' > docs/index.html"
+    "demo": "du -a node_modules/ | awk '{print $1*512 \"\t\" $2}' | node build/src/cli.js --title 'node_modules for webtreemap' > docs/index.html"
```

Comment:
So gross!  But so it goes.  I wonder if there's some alternative to du on OS X that shows bytes?

Reply:
This is the closest I can find:

    find . -type f -ls | awk '{ print $7, $11 }'

Does that work on Linux? It feels fragile.